lambda/MAS/Services/CloudFrontService.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Amazon.CloudFront;
+using Amazon.CloudFront.Model;
+using MAS.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MAS.Services
+{
+    public interface ICloudFrontService
+    {
+        Task<CreateInvalidationResponse> InvalidateCacheAsync(List<string> paths);
```

Comment:
I think this should be part of the staticcontentservice: so far, the controllers don't anything specific about s3/cloudfront